config/index.js

```diff
@@ -493,6 +493,57 @@ var conf = convict({
       default: '.+@mozilla\\.com$',
       env: 'LASTACCESSTIME_UPDATES_EMAIL_ADDRESSES'
     }
+  },
+  signinUnblock: {
+    codeLength: {
+      doc: 'Number of base36 digits to make up an unblockCode',
+      default: 8,
+      env: 'SIGNIN_UNBLOCK_CODE_LENGTH'
+    },
+    codeLifetime: {
+      doc: 'How long an unblockCode should be valid for',
+      format: 'duration',
+      default: '1 hour',
+      env: 'SIGNIN_UNBLOCK_CODE_LIFETIME'
+    },
+    enabled: {
+      default: true
+    },
+    allowEmailRegex: {
+      doc: 'If feature enabled, allow sign-in unblock for email addresses matching this regex.',
+      format: Array,
+      default: [
+        '.+@mozilla\.com$'
```

Comment:
As this is a string, the backslash will not make it into the parsed regex because it's escaping the period. Needs to be a `\\` to escape the backslash.